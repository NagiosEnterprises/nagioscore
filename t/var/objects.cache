########################################
#       NAGIOS OBJECT CACHE FILE
#
# THIS FILE IS AUTOMATICALLY GENERATED
# BY NAGIOS.  DO NOT MODIFY THIS FILE!
#
# Created: Thu May 14 23:56:31 2009
########################################

define timeperiod {
	timeperiod_name	none
	alias	Nothing
	}

define command {
	command_name	check_me
	command_line	/usr/local/nagios/libexec/check_me
	}

define command {
	command_name	notify-none
	command_line	/usr/local/nagios/notifications/notify-none
	}

define contactgroup {
	contactgroup_name	causetestfailure
	alias	This causes a test failure by having a comma separated list before the empty contactgroup
	members	second,nagiosadmin
	}

define contactgroup {
	contactgroup_name	normalhostcontactgroup
	alias	This contact group is assigned to a host normally
	members	second
	}

define contactgroup {
	contactgroup_name	empty
	alias	No members defined - this should pass validation
	}

define contact {
	contact_name	nagiosadmin
	service_notification_period	none
	host_notification_period	none
	service_notification_options	w,u,c,r,f,s
	host_notification_options	d,u,r,f,s
	service_notification_commands	notify-none
	host_notification_commands	notify-none
	host_notifications_enabled	0
	service_notifications_enabled	0
	can_submit_commands	1
	retain_status_information	1
	retain_nonstatus_information	1
	}

define contact {
	contact_name	second
	service_notification_period	none
	host_notification_period	none
	service_notification_options	w,u,c,r,f,s
	host_notification_options	d,u,r,f,s
	service_notification_commands	notify-none
	host_notification_commands	notify-none
	host_notifications_enabled	0
	service_notifications_enabled	0
	can_submit_commands	1
	retain_status_information	1
	retain_nonstatus_information	1
	}

define host {
	host_name	host1
	alias	host1 test
	address	192.168.1.1
	check_period	none
	contacts	nagiosadmin
	contact_groups normalhostcontactgroup
	notification_period	none
	initial_state	o
	check_interval	5.000000
	retry_interval	1.000000
	max_check_attempts	2
	active_checks_enabled	1
	passive_checks_enabled	1
	obsess_over_host	1
	event_handler_enabled	1
	low_flap_threshold	0.000000
	high_flap_threshold	0.000000
	flap_detection_enabled	1
	flap_detection_options	o,d,u
	freshness_threshold	0
	check_freshness	0
	notification_options	d,u,r,f,s
	notifications_enabled	1
	notification_interval	60.000000
	first_notification_delay	0.000000
	stalking_options	n
	stalking_notify	0
	process_perf_data	1
	retain_status_information	1
	retain_nonstatus_information	1
	}

define host {
	host_name	host2
	alias	host2 test
	address	192.168.1.2
	check_period	none
	contacts	nagiosadmin
	notification_period	none
	initial_state	o
	check_interval	5.000000
	retry_interval	1.000000
	max_check_attempts	2
	active_checks_enabled	1
	passive_checks_enabled	1
	obsess_over_host	1
	event_handler_enabled	1
	low_flap_threshold	0.000000
	high_flap_threshold	0.000000
	flap_detection_enabled	1
	flap_detection_options	o,d,u
	freshness_threshold	0
	check_freshness	0
	notification_options	d,u,r,f,s
	notifications_enabled	1
	notification_interval	60.000000
	first_notification_delay	0.000000
	stalking_options	n
	stalking_notify	0
	process_perf_data	1
	retain_status_information	1
	retain_nonstatus_information	1
	}

define service {
	host_name	host1
	service_description	Dummy service
	check_period	none
	check_command	check_me
	contacts	nagiosadmin
	contact_groups normalhostcontactgroup
	notification_period	none
	initial_state	o
	check_interval	32.000000
	retry_interval	1.000000
	max_check_attempts	3
	is_volatile	0
	parallelize_check	1
	active_checks_enabled	1
	passive_checks_enabled	1
	obsess_over_service	1
	event_handler_enabled	1
	low_flap_threshold	0.000000
	high_flap_threshold	0.000000
	flap_detection_enabled	1
	flap_detection_options	o,w,u,c
	freshness_threshold	0
	check_freshness	0
	notification_options	u,w,c,r,f,s
	notifications_enabled	1
	notification_interval	60.000000
	first_notification_delay	0.000000
	stalking_options	n
	stalking_notify	0
	process_perf_data	1
	retain_status_information	1
	retain_nonstatus_information	1
	}

define service {
	host_name	host2
	service_description	Dummy service
	check_period	none
	check_command	check_me
	contacts	nagiosadmin
	notification_period	none
	initial_state	o
	check_interval	32.000000
	retry_interval	1.000000
	max_check_attempts	3
	is_volatile	0
	parallelize_check	1
	active_checks_enabled	1
	passive_checks_enabled	1
	obsess_over_service	1
	event_handler_enabled	1
	low_flap_threshold	0.000000
	high_flap_threshold	0.000000
	flap_detection_enabled	1
	flap_detection_options	o,w,u,c
	freshness_threshold	0
	check_freshness	0
	notification_options	u,w,c,r,f,s
	notifications_enabled	1
	notification_interval	60.000000
	first_notification_delay	0.000000
	stalking_options	n
	stalking_notify	0
	process_perf_data	1
	retain_status_information	1
	retain_nonstatus_information	1
	}

