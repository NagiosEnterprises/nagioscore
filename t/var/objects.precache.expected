########################################
#       NAGIOS OBJECT CACHE FILE
#
# THIS FILE IS AUTOMATICALLY GENERATED
# BY NAGIOS.  DO NOT MODIFY THIS FILE!
#
########################################

define timeperiod {
	timeperiod_name	none
	alias	Nothing
	}

define command {
	command_name	check_me
	command_line	/usr/local/nagios/libexec/check_me
	}

define command {
	command_name	multiple_continuation_lines_with_spaces_intermingled
	command_line	check_nrpe_arg!30!check_fs_ping!/mnt/account-p,/mnt/prepro-p,/mnt/webapp-ssl,/mnt/rollout-p
	}

define command {
	command_name	notify-none
	command_line	/usr/local/nagios/notifications/notify-none
	}

define command {
	command_name	set_to_stale
	command_line	/usr/local/nagios/libexec/set_to_stale
	}

define command {
	command_name	with_continuation_lines
	command_line	$USER1$/check_foo onetwo
	}

define contactgroup {
	contactgroup_name	causetestfailure
	alias	This causes a test failure by having a comma separated list before the empty contactgroup
	members	second,nagiosadmin
	}

define contactgroup {
	contactgroup_name	empty
	alias	No members defined - this should pass validation
	}

define hostgroup {
	hostgroup_name	hosts-with-master-service
	alias	Hosts running a master service
	members	host3
	}

define servicegroup {
	servicegroup_name	services-depending-on-master-service
	alias	Servicegroup for services depending on a "master" service on the same host
	members	host3,dependent-service
	}

define contact {
	contact_name	nagiosadmin
	alias	nagiosadmin
	service_notification_period	none
	host_notification_period	none
	service_notification_options	r,w,u,c,f,s
	host_notification_options	r,d,u,f,s
	service_notification_commands	notify-none
	host_notification_commands	notify-none
	minimum_importance	0
	host_notifications_enabled	0
	service_notifications_enabled	0
	can_submit_commands	1
	retain_status_information	1
	retain_nonstatus_information	1
	}

define contact {
	contact_name	second
	alias	second
	service_notification_period	none
	host_notification_period	none
	service_notification_options	r,w,u,c,f,s
	host_notification_options	r,d,u,f,s
	service_notification_commands	notify-none
	host_notification_commands	notify-none
	minimum_importance	0
	host_notifications_enabled	0
	service_notifications_enabled	0
	can_submit_commands	1
	retain_status_information	1
	retain_nonstatus_information	1
	}

define host {
	host_name	host1
	alias	host1 test
	address	192.168.1.1
	check_period	none
	contacts	nagiosadmin
	notification_period	none
	initial_state	o
	importance	0
	check_interval	5.000000
	retry_interval	1.000000
	max_check_attempts	2
	active_checks_enabled	1
	passive_checks_enabled	1
	obsess	1
	event_handler_enabled	1
	low_flap_threshold	0.000000
	high_flap_threshold	0.000000
	flap_detection_enabled	1
	flap_detection_options	a
	freshness_threshold	0
	check_freshness	0
	notification_options	a
	notifications_enabled	1
	notification_interval	60.000000
	first_notification_delay	0.000000
	stalking_options	n
	process_perf_data	1
	retain_status_information	1
	retain_nonstatus_information	1
	}

define host {
	host_name	host2
	alias	host2 test
	address	192.168.2.2
	check_period	none
	contacts	nagiosadmin
	notification_period	none
	initial_state	o
	importance	0
	check_interval	5.000000
	retry_interval	1.000000
	max_check_attempts	2
	active_checks_enabled	1
	passive_checks_enabled	1
	obsess	1
	event_handler_enabled	1
	low_flap_threshold	0.000000
	high_flap_threshold	0.000000
	flap_detection_enabled	1
	flap_detection_options	a
	freshness_threshold	0
	check_freshness	0
	notification_options	a
	notifications_enabled	1
	notification_interval	60.000000
	first_notification_delay	0.000000
	stalking_options	n
	process_perf_data	1
	retain_status_information	1
	retain_nonstatus_information	1
	}

define host {
	host_name	host3
	alias	host3 test
	address	192.168.2.3
	check_period	none
	contacts	nagiosadmin
	notification_period	none
	initial_state	o
	importance	0
	check_interval	5.000000
	retry_interval	1.000000
	max_check_attempts	2
	active_checks_enabled	1
	passive_checks_enabled	1
	obsess	1
	event_handler_enabled	1
	low_flap_threshold	0.000000
	high_flap_threshold	0.000000
	flap_detection_enabled	1
	flap_detection_options	a
	freshness_threshold	0
	check_freshness	0
	notification_options	a
	notifications_enabled	1
	notification_interval	60.000000
	first_notification_delay	0.000000
	stalking_options	n
	process_perf_data	1
	retain_status_information	1
	retain_nonstatus_information	1
	}

define service {
	host_name	host1
	service_description	Dummy service
	check_period	none
	check_command	check_me
	contacts	nagiosadmin
	notification_period	none
	initial_state	o
	importance	0
	check_interval	32.000000
	retry_interval	1.000000
	max_check_attempts	3
	is_volatile	0
	parallelize_check	1
	active_checks_enabled	1
	passive_checks_enabled	1
	obsess	1
	event_handler_enabled	1
	low_flap_threshold	0.000000
	high_flap_threshold	0.000000
	flap_detection_enabled	1
	flap_detection_options	a
	freshness_threshold	0
	check_freshness	0
	notification_options	a
	notifications_enabled	1
	notification_interval	60.000000
	first_notification_delay	0.000000
	stalking_options	n
	process_perf_data	1
	retain_status_information	1
	retain_nonstatus_information	1
	}

define service {
	host_name	host1
	service_description	Uses important check command
	check_period	none
	check_command	set_to_stale
	contacts	nagiosadmin
	notification_period	none
	initial_state	o
	importance	0
	check_interval	15.000000
	retry_interval	1.000000
	max_check_attempts	5
	is_volatile	0
	parallelize_check	1
	active_checks_enabled	1
	passive_checks_enabled	1
	obsess	1
	event_handler_enabled	1
	low_flap_threshold	0.000000
	high_flap_threshold	0.000000
	flap_detection_enabled	1
	flap_detection_options	a
	freshness_threshold	0
	check_freshness	0
	notification_options	a
	notifications_enabled	1
	notification_interval	65.000000
	first_notification_delay	0.000000
	stalking_options	n
	process_perf_data	1
	retain_status_information	1
	retain_nonstatus_information	1
	}

define service {
	host_name	host2
	service_description	Uses important check command
	check_period	none
	check_command	set_to_stale
	contacts	nagiosadmin
	notification_period	none
	initial_state	o
	importance	0
	check_interval	15.000000
	retry_interval	1.000000
	max_check_attempts	5
	is_volatile	0
	parallelize_check	1
	active_checks_enabled	1
	passive_checks_enabled	1
	obsess	1
	event_handler_enabled	1
	low_flap_threshold	0.000000
	high_flap_threshold	0.000000
	flap_detection_enabled	1
	flap_detection_options	a
	freshness_threshold	0
	check_freshness	0
	notification_options	a
	notifications_enabled	1
	notification_interval	65.000000
	first_notification_delay	0.000000
	stalking_options	n
	process_perf_data	1
	retain_status_information	1
	retain_nonstatus_information	1
	}

define service {
	host_name	host3
	service_description	dependent-service
	check_period	none
	check_command	check_me!dependent service
	contacts	nagiosadmin
	notification_period	none
	initial_state	o
	importance	0
	check_interval	15.000000
	retry_interval	1.000000
	max_check_attempts	5
	is_volatile	0
	parallelize_check	1
	active_checks_enabled	1
	passive_checks_enabled	1
	obsess	1
	event_handler_enabled	1
	low_flap_threshold	0.000000
	high_flap_threshold	0.000000
	flap_detection_enabled	1
	flap_detection_options	a
	freshness_threshold	0
	check_freshness	0
	notification_options	a
	notifications_enabled	1
	notification_interval	65.000000
	first_notification_delay	0.000000
	stalking_options	n
	process_perf_data	1
	retain_status_information	1
	retain_nonstatus_information	1
	}

define service {
	host_name	host3
	service_description	master-service
	check_period	none
	check_command	check_me!master service
	contacts	nagiosadmin
	notification_period	none
	initial_state	o
	importance	0
	check_interval	15.000000
	retry_interval	1.000000
	max_check_attempts	5
	is_volatile	0
	parallelize_check	1
	active_checks_enabled	1
	passive_checks_enabled	1
	obsess	1
	event_handler_enabled	1
	low_flap_threshold	0.000000
	high_flap_threshold	0.000000
	flap_detection_enabled	1
	flap_detection_options	a
	freshness_threshold	0
	check_freshness	0
	notification_options	a
	notifications_enabled	1
	notification_interval	65.000000
	first_notification_delay	0.000000
	stalking_options	n
	process_perf_data	1
	retain_status_information	1
	retain_nonstatus_information	1
	}

define servicedependency {
	host_name	host3
	service_description	master-service
	dependent_host_name	host3
	dependent_service_description	dependent-service
	inherits_parent	0
	notification_failure_options	u,c
	}

define serviceescalation {
	host_name	host1
	service_description	Uses important check command
	first_notification	-2
	last_notification	-2
	notification_interval	65.000000
	escalation_period	none
	escalation_options	a
	contacts	nagiosadmin
	}

define serviceescalation {
	host_name	host2
	service_description	Uses important check command
	first_notification	-2
	last_notification	-2
	notification_interval	65.000000
	escalation_period	none
	escalation_options	a
	contacts	nagiosadmin
	}

define serviceescalation {
	host_name	host3
	service_description	dependent-service
	first_notification	-2
	last_notification	-2
	notification_interval	65.000000
	escalation_period	none
	escalation_options	a
	contacts	nagiosadmin
	}

define serviceescalation {
	host_name	host3
	service_description	master-service
	first_notification	-2
	last_notification	-2
	notification_interval	65.000000
	escalation_period	none
	escalation_options	a
	contacts	nagiosadmin
	}

